app/src/app/[lng]/auth/signup/page.tsx

```diff
@@ -116,7 +116,12 @@ export default function Signup({
             {errors.name && errors.name.message}
           </FormErrorMessage>
         </FormControl>
-        <EmailInput register={register} error={errors.email} t={t} />
+        <EmailInput
+          disabled={false}
```

Comment:
Why the disabled={false} attribute here? That should be the default?

Reply:
I see, it's a custom attribute. You can make it an optional property in EmailInput and set the default value to false.